pkg/metrics/metrics.go

```diff
@@ -9,42 +9,43 @@ func init() {
 }
 
 var (
-	M_Instance_Start                     Counter
```

Comment:
Seems like a tabs vs space issue. What editor are you using? Any chance you can find a plugin for editorconfig? https://github.com/grafana/grafana/blob/master/.editorconfig http://editorconfig.org/